Abstract Syntax and Semantics Document/index.html

```diff
@@ -500,53 +500,159 @@
+      <p>Given any two timestamped graphs, <code>TSG1</code> and <code>TSG2</code>, we say that <code>TSG2</code> <em>covers</em> <code>TSG1</code> 
+      (denoted <code>TSG1 &le; TSG2</code>) if and only if <code>TSG1</code> and <code>TSG2</code> have the same timestamp predicate <code>P</code>
+        and the timestamps, <code>t1</code> and <code>t2</code> resp., satisfy <code>t1 &le;<sub>P</sub> t2</code>, where 
+        <code>&le;<sub>P</sub></code> is the temporal partial order 
+        associated with the timestamp predicate <code>P</code>.
+    </p>
       
-      <p class="note">In the following, we take the case where temporal entities in the stream are time instants.</p>
+      <p class="ednote">The definition of cover, and the following discussion of it, could probably be moved to follow shortly after the definition of timestamped graph so that
+      this concept could be used in other definition.</p>
       
-      <p>Open Issue: <a href="https://github.com/streamreasoning/RSP-QL/issues/18">Issue 18</a></p>
-    </blockquote>
-    
-    <p>A <em>window function</em> <code>w<sub>ι</sub></code> of type <code>ι</code> takes as input a stream <code>S</code>, a predicate <code>p</code>, a time instant <code>t</code>, called the reference time point, and a vector of window parameters <code>x</code> for type <code>ι</code> and returns a substream <code>S'</code> of <code>S</code> that contains only timestamped graphs associated with <code>p</code> and timestamps valid at <code>t</code> according <code>ι</code> and <code>x</code>.</p>
+      <p class="note">The relation <code>&le;</code> between timestamped graphs is a preorder (a reflexive, transitive binary relation).
+        It is not a partial order because it doesn't have the antisymmetry property (<code>a &le; b</code> and <code>b &le; a</code> implies <code>a = b</code>.) 
+      </p>
+      
+      <aside class="example highlight" title="Distinct Timestamped Graphs That Cover Each Other">
+        <p>Two timestamped graphs that have the same timestamp predicate and the same timestamp but different named graphs
+          cover each other, but are not equal.
+          This demonstrates that the "cover" relation is not antisymmetric.
+        </p>
+      </aside>
+      
+      <p>A <em>window</em> <code>S'</code> of an RDF stream <code>S</code> is a substream of <code>S</code> such that 
+        if <code>S'(i)</code> and <code>S'(j)</code> are two elements of <code>S'</code>
+          and <code>S'(i) &le; S'(j)</code>, and further if <code>S'(i) &le; S(k) &le; S'(j)</code> for some element <code>S(k)</code> of <code>S</code>, 
+            then <code>S(k)</code> is an element of <code>S'</code>.      
+      </p>
+      <p class="note">Informally, a window is a temporally-contiguous selection from the original stream, without gaps.</p>
+      
+      <p class="ednote">The definition of window should be moved to the substream section.</p>
```

Comment:
As the ednote says, the plan is to move the definition of window into the substream section, following the definition fo substream.